app/test_main.py

```diff
@@ -2,0 +1,17 @@
+from unittest import mock
+
+from app.main import can_access_google_page
+
+
+@mock.patch("app.main.valid_google_url")
+def test_is_valid_google_url(mocked_valid_google_url: mock.MagicMock) -> None:
+    can_access_google_page("https://www.google.com/")
```

Comment:
The `can_access_google_page` function is called with a hardcoded URL. Ensure that this is the intended behavior for the test and that the function handles this input correctly.